thyrel-api/thyrel-api/Controllers/ElementController.cs

```diff
@@ -131,5 +133,60 @@ public async Task<ActionResult<ElementStepDto>> GetCurrent()
 
             return await new ElementDataProvider(_context).GetCurrentElement(player.Id);
         }
+
+        /// <summary>
+        /// call this endpoint to add a new emojiReaction to an element
+        /// </summary>
+        /// <param name="elementId"></param>
+        /// <param name="emojiReaction"></param>
+        /// <returns></returns>
+        [HttpPost("{elementId}/reaction")]
+        public async Task<ActionResult> AddReaction(int elementId, [FromBody] EmojiReaction emojiReaction)
+        {
+            var player = await AuthorizationHandler.CheckAuthorization(HttpContext, _context);
+            if (player?.RoomId == null && isElementInLastSession(elementId, (int)player.RoomId)) return Unauthorized();
+
+            var reaction = await _reactionDataProvider.AddReaction(player.Id, elementId, emojiReaction);
+
+            await _websocketHandler.SendMessageToSockets(
+                JsonBase.Serialize(
+                    new EmojiReactionWebSocketEventJson(player.Id, elementId, reaction.Emoji)), player.RoomId);
+
+            return Ok(200);
```

Comment:
```suggestion
            return NoContent();
```
`Ok` => return 200
`Ok(1)` => return 200 avec comme body un nombre